source/lex.tex

```diff
@@ -668,8 +674,8 @@
 \tcode{audit}           \\
 \tcode{axiom}           \\
 \columnbreak
-\tcode{import}          \\
 \tcode{final}           \\
+\tcode{import}          \\
 \columnbreak
 \tcode{module}          \\
 \tcode{override}        \\
```

Comment:
Can we use \keyword in this table instead of adding explicit index entries?

Reply:
(Likewise in the keyword table below; those should get index entries for the keyword too, I think.)

Reply:
I've also added the alternative operator spellings such as "and" and "bitand". They are sufficiently keyword-like to warrant an index entry, I believe.

Reply:
Ah, the \keyword macro is only defined within "bnfbase" environments. We need it outside, too.

Reply:
Fixed.